lib/utils.js

```diff
@@ -120,3 +120,19 @@ exports.parseDate = function(dateString) {
 exports.toISO8601String = function(date) {
   return new Date(date).toISOString().replace(/[-:]|\.\d+/g, "");
 };
+
+/**
+ * Reads a request body stream to a string.
+ *
+ * @param ctx
+ * @returns {Promise<string>}
```

Comment:
This is actually `Promise<void>`, we're not returning anything. You can also give `ctx` a type of `Koa.Context` if you'd like (there's no TS compatibility yet so it doesn't matter too much to me).